compareSvsTSVs

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env scala
```

Comment:
took us 5 mins to figure out: This is a different name than the file created in commit "Added comparison script for Sophia result directories. No complete co… " <- not a good idea in our opinion